crates/typst-library/src/layout/fr.rs

```diff
@@ -8,15 +8,13 @@ use typst_utils::{Numeric, Scalar};
 use crate::foundations::{repr, ty, Repr};
 use crate::layout::Abs;
 
-/// Defines how the remaining space in a layout is distributed.
+/// レイアウトに残されたスペースをどう分配するかを定義。
 ///
-/// Each fractionally sized element gets space based on the ratio of its
-/// fraction to the sum of all fractions.
+/// 比率の大きさを持つ各要素に割り当てられるスペースは、全ての比率の和に対する割合に基づいて決まります。
```

Comment:
文法的には、`Each fractionally sized element`が主語、`gets`が動詞、`space`以降が目的語かと思います。たとえば以下のような訳はいかがでしょうか？

- fractionは頻出するわけではないと思いますので、用語集には載せる必要は無いと思いますが、カッコ書きで補足を入れてみました。
- `Each fractionally sized element`がそのまま訳しづらかったのと、主語が重たく感じたため、条件として切り出すような意訳をしました。
- 直訳では無生物主語能動態ですが、受動態に意訳しています。

```suggestion
/// それぞれの要素が比率（fraction）で指定されている場合、その要素は、すべての割合の合計に対する自身の比率に応じてスペースを割り当てられます。
```